### ISSUE TITLE
use current path instead of module path for config spec first

### ISSUE THREAD

**Issue by ivnovi** [bundyfx/vamp#20]
It would make sense to store config/spec files separate from the module or even have multiple environments if you need without duplicating module.

Currently $PsScriptRoot points to module directory it seems. may be use $pwd.path as default option instead? and if it doesn't contain anything then use module path (or not).

**Comment by flynnCoolblue** [comment]
Makes sense. Simple enough change, feel free to make it and shoot through the PR :)
